docs/dashboards/chart-builder.md

```diff
@@ -176,3 +190,18 @@ Variables can be created in Report options "Manage variables".
 
 ![Manage Variables](../images/dashboards/variables.png)
 
+
+## Style
+
+In the style tab, you configure what the chart looks like. You can select bar, line, area and point 
+charts. In addition to chart type, you can choose a color from the palette and group color scheme. The main
+goal of a group color scheme is to determine the colors of the series from the same group. When you group metric by
+`os.host` as a result you get a couple of series. Each series can get a different color.
+
+![Chart builder styles](../images/dashboards/chart-builder-styles.png)
+
+A very important feature in the styles tab is "Stacked" option. When you select the checkbox data
```

Comment:
A very important feature in the styles tab is "Stacked" option. When you select the checkbox data
=>
A very important feature in the styles tab is the "Stacked" option. When you select the checkbox data